### PR TITLE
Feature/quiz picker baseview

### PR DESCRIPTION
**Pull request by Derstilon**
image
I've created base functionality for quiz creation using a transfer list and text field

### CODE REVIEW

File: src/lecturer/@types/types.d.ts [hunk 1]
```diff
@@ -8,3 +8,19 @@ interface Student {
     surname: string;
     nick: string;
 }
+
+interface Answer {
```
Comment (rozchlastywacz) [inline src/lecturer/@types/types.d.ts:12]: image

File: src/lecturer/components/pickQuizView/PickQuizView.tsx [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+    const handleToggle = (value: number) => () => {
+        const currentIndex = checked.indexOf(value);
+        const newChecked = [...checked];
+
+        if (currentIndex === -1) {
+            newChecked.push(value);
+        } else {
+            newChecked.splice(currentIndex, 1);
+        }
+
```
Comment (rozchlastywacz) [inline src/lecturer/components/pickQuizView/PickQuizView.tsx:139]: is this blank line needed?

**Review by rozchlastywacz — APPROVED** [review]
You could make more files that export certain function, but aside from that everything looks great

**Review by michal367 — APPROVED** [review]
OK

**Pull request merged**